backend/.env.example

```
@@ -0,0 +1,589 @@
+DJANGO_ALLOWED_HOSTS=localhost,127.0.0.1,retailtradescannet.com,api.retailtradescannet.com,tradescanpro.com,www.tradescanpro.com
+CSRF_TRUSTED_ORIGINS=https://api.retailtradescannet.com,https://tradescanpro.com,https://www.tradescanpro.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CSRF_TRUSTED_ORIGINS key should go before the DJANGO_ALLOWED_HOSTS key

(UnorderedKey)